virtual/qemu/BUILD

```diff
@@ -9,20 +9,28 @@ in_depends $MODULE esound && AUDIO_LIST+=",esd"
 in_depends $MODULE pulseaudio && AUDIO_LIST+=",pa"
 in_depends $MODULE SDL && AUDIO_LIST+=",sdl"
 
+if [ "$ENABLE_KVM" == "y" ]; then
```

Comment:
You should just append --enable-virtfs to the ENABLE_KVM mquery in CONFIGURE, cleaner that way :)